docs/services/vision/obstacles_distance.md

```diff
@@ -139,3 +139,13 @@ To learn more about how to use segmentation, see the [Go SDK docs](https://pkg.g
 {{% alert title="Tip" color="tip" %}}
 To see more code examples of how to use Viam's vision service, see [our example repo](https://github.com/viamrobotics/vision-service-examples).
```

Comment:
```suggestion
To see more code examples of how to use Viam's Vision service, see [our example repo](https://github.com/viamrobotics/vision-service-examples).
```
if we do decide to capitalize should keep consistent

Reply:
no - rule is not to capitalize, so the capitalization is wrong